composables/transaction/mintCollection/useNewCollectionId.ts

```diff
@@ -1,36 +1,3 @@
-import { getRandomValues } from '@/components/unique/utils'
-import { mapToId } from '@/utils/mappers'
-import { unwrapSafe } from '@/utils/uniquery'
-
-export function useNewCollectionId() {
```

Comment:
nit: let's move to separate PR regarding remove unused stuff

Reply:
sure